backend/plugin/db/pg/pg.go

```diff
@@ -646,7 +685,7 @@ func (d *Driver) GetCurrentDatabaseOwner(ctx context.Context) (string, error) {
 }
 
 // QueryConn queries a SQL statement in a given connection.
-func (*Driver) QueryConn(ctx context.Context, conn *sql.Conn, statement string, queryContext db.QueryContext) ([]*v1pb.QueryResult, error) {
+func (d *Driver) QueryConn(ctx context.Context, conn *sql.Conn, statement string, queryContext db.QueryContext) ([]*v1pb.QueryResult, error) {
```

Comment:
is it possible to conn.(*pgconn.PgConn)?

Reply:
Impossible. The conn is not a Golang Interface.

Reply:
Using conn.Raw makes it possible. e.g.
```golang
	conn, err := s.DB.Conn(ctx)
	if err != nil {
		return err
	}
	defer conn.Close()

	var res pgconn.CommandTag
	err = conn.Raw(func(driverConn interface{}) error {
		pgxConn := driverConn.(*stdlib.Conn).Conn()
		res, err = pgxConn.PgConn().CopyFrom(ctx, &buf, "COPY csv_test FROM STDIN (FORMAT csv)")
		if err != nil {
			return err
		}
		return nil
	})
```

Reply:
https://github.com/bytebase/bytebase/blob/2060bb74fec47c7d2af48161156b5ee055b35d1b/backend/plugin/db/pg/pg.go#L456-L458

Reply:
We cannot change the connection config after it's been created.